pinc/dpsql.inc

```diff
@@ -24,11 +24,7 @@ function dpsql_dump_themed_query($query, $start_at = 0, $show_rank = DPSQL_NO_RA
 
 function dpsql_query($query)
```

Comment:
I was wondering why you did it like this but I see mysqli_query is used in noncvs.